level1a/handlers/level1a.py

```diff
@@ -22,11 +22,17 @@
 OFFSET_FACTOR = 2
 ORBIT_FREQUENCY = 0.1
 ATTITUDE_FREQUENCY = 1.
+HTR_FREQUENCY = 0.1
```

Comment:
Where is this from?

Reply:
Also: consider adding the unit after as a comment; I thing you will find that it is not a frequency but an interval... ;)  (No, I don't think you need to change the name, but something saying that it is seconds (if that is what it is) would be good.)

Reply:
I checked the file and it's 10 seconds between rows, so 0.1 Hz. Same goes for `ORBIT` and `ATTITUDE`. But I'll add add a comment with unit

Reply:
Or I can flip it around to be the interval if that's better